third_party/multiarch/qemu-user-static/register/qemu-binfmt-conf.sh

```diff
@@ -0,0 +1,367 @@
+#!/bin/sh
```

Comment:
`third_party/multiarch/qemu-user-static` is required because we need to run `apt` binary to update packages in debian-base image.